delta_rest_client/delta_rest_client.py

```diff
@@ -203,3 +185,29 @@ def round_by_tick_size(price, tick_size, floor_or_ceil=None):
         return price - remainder + tick_size
     else:
         return price - remainder
+
+def generate_signature(secret, message):
+    message = bytes(message, 'utf-8')
+    secret = bytes(secret, 'utf-8')
+    hash = hmac.new(secret, message, hashlib.sha256)
+    return hash.hexdigest()
+
+def get_time_stamp():
+    d = datetime.datetime.utcnow()
+    epoch = datetime.datetime(1970,1,1)
+    return str(int((d - epoch).total_seconds()))
+
+def query_string(query):
+    if query == None:
+        return ''
+    else:
+        query_string = '?'
+        for key, value in query.items():
+            query_string += key + '=' + str(value) + '&'
+        return query_string[:-1]
```

Comment:
Nopes. This code is too ugly. Please collect it in an array and join it with '&'.